spec/server/ServerRendererSpec.js

```diff
@@ -181,7 +181,7 @@ describe("ServerRenderer", function() {
     describe("when layout does NOT have a body", function() {
 
         beforeEach(function() {
-            layout.el.innerHTML = "<html><head></head></html>";
+            layout.$("body").remove();
```

Comment:
If the user specifies a template without a body, it will be automatically inserted somewhere at the JSDOM level. I think this test is probably not necessary.